brain_proxy/brain_proxy.py

```
@@ -376,27 +613,30 @@ async def _store_mem(memories: List[Any]):
                     self._log(f"Error processing memory: {e}")
             
             if docs:
-                self._log(f"Storing {len(docs)} memories for tenant {tenant}")
+                self._log(f"Storing {len(docs)} memories for tenant {mem_key}")
                 await vec.add_documents(docs)
                 self._log(f"Successfully stored memories")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)